draft-ietf-quic-load-balancers.md

```diff
@@ -756,6 +751,45 @@ the servers.
 Note that future versions of QUIC might not have Retry packets, require
 different information in Retry, or use different packet type indicators.
 
+### Considerations for Non-Initial Packets
+
+Initial Packets are especially effective at consuming server resources
+because they cause the server to create connection state. Even when mitigating
+this load with Retry Packets, the act of validating an Initial Token and sending
+a Retry Packet is more expensive than the response to a non-Initial packet with
+an unknown Connection ID: simply dropping it.
```

Comment:
Do you mean a non-Initial & non-1-RTT packet here? You respond to an unknown 1-RTT packet with a stateless retry.

Reply:
Oh, good point.